libs/data-access/src/lib/concept/api.ts

```diff
@@ -0,0 +1,12 @@
+export const getConceptsForCatalog = async (
+  catalog_id: string,
+  access_token: string
+) => {
+  const url = `${process.env.NEXT_PUBLIC_CONCEPT_REGISTRATION_API}/begreper?orgNummer=${catalog_id}`;
```

Comment:
```suggestion
  const url = `${process.env.NEXT_PUBLIC_CONCEPT_REGISTRATION_API}/begreper/search?orgNummer=${catalog_id}`;
```

Det er sikkert greit å alltid kun bruke søk-endepunktet. Det er da dog snakk om POST, ikke GET, og den trenger en body som inneholder feltet `query`. Du kan se på denne commiten jeff lagde til POC'en vi satte opp: https://github.com/Informasjonsforvaltning/concept-catalog-gui/commit/8c9cf4f55329556253e8be35c3daeeff7304a77f

Reply:
Ja, smart! Skal gjøre det.